### PR TITLE
[KHWEBA-75] Transaction dialog

### PR DESCRIPTION
**Pull request by gift-a**
image

### CODE REVIEW

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

**Review by OlgaOrlova — DISMISSED** [review]

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

File: src/app/transactions/transactions.component.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.add-transaction-dialog-activation {
```
Comment (ihor7016) [inline src/app/transactions/transactions.component.scss]: we've agreed to use BEM

**Comment by ihor7016** [comment]
why yellow color for button and radios? please...

**Comment by gift-a** [comment]
there are 4 color themes:
`    deeppurple-amber.css`
`    indigo-pink.css`
`    pink-bluegrey.css`
`    purple-green.css`
first one is the best

File: src/app/dialog/dialog.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.dialog {
```
Comment (ihor7016) [inline src/app/dialog/dialog.scss]: could you extract style-only components into folder named `app/styles/` . e.g `app/styles/dialog.scss`

File: src/app/transaction-dialog/transaction-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { Component, Inject } from "@angular/core";
```
Comment (ihor7016) [inline src/app/transaction-dialog/transaction-dialog.component.ts]: why this is not inside `transactions` folder?

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**